pact-lsp/Pact/Core/LanguageServer.hs

```diff
@@ -205,7 +204,6 @@ sendDiagnostics nuri mv content = liftIO runPact >>= \case
     runPact = do
       let file = fromMaybe "<local>" $ uriToFilePath (fromNormalizedUri nuri)
       pdb <- mockPactDb serialisePact_repl_spaninfo
-      gasRef <- newIORef (Gas 0)
```

Comment:
Just to confirm — we're using `eeGasRef` for accounting now, right?

Reply:
Yes